integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -52,11 +52,14 @@ def transform_provider_id(self, content_metadata_item):  # pylint: disable=unuse
         """
         return self.enterprise_configuration.provider_id
 
-    def transform_status(self, content_metadata_item):  # pylint: disable=unused-argument
+    def transform_status(self, content_metadata_item):
```

Comment:
Delete is sent to SAP by setting `status` to `INACTIVE`. This code needs to be aware of that and not just blindly set an `ACTIVE` status every time.